docker/hub-config/config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "serverName": "hub",
+  "port": 3000,
+  "driver": "disk",
+  "bucket": "hub",
+  "readURL": "http://localhost:80/hub/",
```

Comment:
This needs to be set to the DNS hostname, the reader prefix, and should be a `https` URL -- so, if I set up gaia.aaron.com to at my VM -- this would need to be:

```
"https://gaia.aaron.com/reader/"
```

Reply:
this will require some changes to the image ignition script. 
it was planned already for the admin API_KEY, so i'll do that and the readURL based on this feedback.

Reply:
addressed in: https://github.com/blockstack/gaia/blob/feature/docker-compose/docker/update_configs.sh

which is called in the gaia-hub service